### DISCUSSION TITLE
宇宙最強創作工具 Omniverse

### DISCUSSION THREAD

**Discussion by ghost** [1101-GS4719-C/Python-Community-GS4719-C#305] [科技新知]
[【CES 2022】NVIDIA 佛心開放元宇宙最強創作工具 Omniverse！「一鍵協作」新功能、超龐大素材庫一次看](https://buzzorange.com/techorange/2022/01/07/nvidia-omniverse/?fbclid=IwAR2BMgL8aJyOgxMVowF2qsVuwIpqffmL1FxdWLHhsFa_ShObKKabnfNnP10)

感覺很酷的東西!而且有個人免費版本

**Comment by 107602505** [comment]
我一開始以為Omniverse 與元宇宙 (Metaverse) 是一樣的概念，上網查才知道它是為開發者提供可以建造元宇宙的工具平台，這個整合平台可以應用在建築、製造、半導體等不同的領域上面，過往這些產業生產、製造、驗證、研發時，都會有不同的軟體與資料格式，Omniverse 的工作就是整合工具與流程，讓每個階段的協同合作，變得更有效率。
這篇報導寫得蠻詳細的~
https://buzzorange.com/techorange/2021/11/19/nvidia-omniverse-factory/

**Comment by EricAgumonJou** [comment]
我簡單地補充一下什麼是Omniverse.
全能宇宙（omniverse）囊括了Universe（宇宙）、Multiverse（多重宇宙、複合型宇宙、多元宇宙、平行宇宙）、Metaverse（超宇宙、位面級宇宙）、Xenoverse（外宇宙、至高宇宙）所有宇宙，比超膜理論更全面且更森羅萬象，包含所有意識形態、空間、法則、維度，包含所有已知的和未知事物的全能核心宇宙。
